operator/controllers/helmrelease_controller_test.go

```diff
@@ -22,6 +22,13 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+type fakeSlack struct{}
+
+func (m *fakeSlack) Send(msg string) error {
+	fmt.Println(msg)
```

Comment:
Printing the messages to stdout makes it difficult to programmatically inspect the sent messages. We could make `fakeSlack` easier to test by removing the IO side-effect and accumulating a `[]string` instead.